tests/distributions/test_stable.py

```diff
@@ -62,3 +63,13 @@ def cdf(x):
 
     stat, pvalue = kstest(sampler, cdf, N=num_samples)
     assert pvalue > 0.1, pvalue
+
+
+@pytest.mark.parametrize("loc", [0, 1, -1, 2, 2])
+@pytest.mark.parametrize("scale", [0.5, 1, 2])
+def test_normal(loc, scale):
+    num_samples = 100000
+    expected = dist.Normal(loc, scale).sample([num_samples])
+    actual = dist.Stable(2, 0, scale * 0.5 ** 0.5, loc).sample([num_samples])
```

Comment:
@fehiepsi `Stable(2, 0, scale * sqrt(2), loc) = Normal(loc, scale)`

Reply:
Yeah, I know about it. I guess you mean `scale / sqrt(2)` here?

Reply:
The test passes as-is. Our implementation agrees with scipy.stats. Wikipedia and scipy.stats disagree by a factor of 1.

Reply:
Hmm, in the implementation, `scale * 0.5 ** 0.5` will be equal to `scale / sqrt(2)`. Similarly, in stable implementation, you divide scale to `sqrt(0.5)`, which is equivalent to multiply with `sqrt(2)`. Both are equivalent IMO. The implementation here agrees with wikipedia, so if scipy is different, probably scipy is wrong (assuming the tests pass)?

Reply:
Yes yes, I simply misread my own code 😊 Thanks for pointing out.